pkg/cmd/server/api/validation/master.go

```diff
@@ -519,9 +519,9 @@ func ValidateRoutingConfig(config api.RoutingConfig) fielderrors.ValidationError
 }
 
 func ValidateAPIServerExtendedArguments(config api.ExtendedArguments) fielderrors.ValidationErrorList {
-	return ValidateExtendedArguments(config, kapp.NewAPIServer().AddFlags)
+	return nil //ValidateExtendedArguments(config, kapp.NewAPIServer().AddFlags)
```

Comment:
> This is just temporary (why this is WIP) until we can split out the upstream code so that depending on the validation here doesn't import kubelet/* or kube-codebase/*. Those changes landed and will either be in the next rebase or the one after that.

Ok.  Edited name to indicate [WIP].  I'd rather hold this until after 1.1.1, so we don't lose validation for master-config.yaml files.